test/extended/operators/images.go

```diff
@@ -0,0 +1,94 @@
+package operators
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+
+	"github.com/openshift/origin/pkg/oc/cli/admin/release"
+	exutil "github.com/openshift/origin/test/extended/util"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+)
+
+var _ = Describe("[Feature:Platform][Smoke] Managed cluster", func() {
+	f := e2e.NewDefaultFramework("operators")
```

Comment:
you don't need a namespace so you can save 30s off your test by setting SkipNamespace.